app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,10 @@
+import camelCase from "lodash/camelCase";
```

Comment:
Don't use `lodash`. In BB we don't use lodash. Either use [Ramda](https://academy.bigbinary.com/learn-react/pages/learn-ramda-js) or create this util function using vanilla JS.